gax/src/main/java/com/google/api/gax/batching/FlowControlSettings.java

```diff
@@ -82,7 +82,16 @@ public static Builder newBuilder() {
   public abstract static class Builder {
     public abstract Builder setMaxOutstandingElementCount(Integer value);
 
-    public abstract Builder setMaxOutstandingRequestBytes(Integer value);
+    public abstract Builder setMaxOutstandingRequestBytes(Long value);
+
+    @Deprecated
+    public Builder setMaxOutstandingRequestBytes(Integer value) {
+      Long l = null;
+      if (value != null) {
+        l = value.longValue();
+      }
+      return setMaxOutstandingRequestBytes(l);
```

Comment:
use a longer variable name than `l`. On this line, it almost looks like you are passing in `1`.